packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -69,6 +69,7 @@ export function getWebpackConfig(bundle: Bundle, worker: WorkerConfig) {
     },
 
     optimization: {
+      usedExports: true,
```

Comment:
have u found any performance impact on dev? In case you have it, maybe we can disable it for dev :)

Reply:
It's actually 20% faster in development for new platform plugins, I assume that is credited to removing `@babel/plugin-transform-modules-commonjs` and `@babel/plugin-syntax-dynamic-import`.

Keep in mind, this is not removing dead code in development, it's just tagging it. It will only be removed once Terser is ran with `optimize=true`